packit_service/constants.py

```diff
@@ -234,3 +234,5 @@ def from_number(number: int):
     "grant `{packit_user}` user `Developer` permissions on the `{namespace}/{repo}`"
     " repository. You can do so [here]({url}/-/project_members)."
 )
+
+DASHBOARD_JOBS_TESTING_FARM = "https://dashboard.packit.dev/jobs/testing-farm-runs"
```

Comment:
```suggestion
DASHBOARD_JOBS_TESTING_FARM = f"https://dashboard{'.stg' if getenv('DEPLOYMENT') == 'stg' else ''}.packit.dev/jobs/testing-farm-runs"
```

Reply:
good point, thanks! I used the existing service config option